app/queries/decidim/assemblies/prioritized_assemblies.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
@deivid-rodriguez This class is redefined to hide child assemblies on `/assemblies` page. Thoughts?

Reply:
Looks good to me, maybe I would explore if there's the possibility of keeping the original implementation and add the extra filter on top of it.

Reply:
I've reviewed the original file and we have nothing to keep there: `Decidim::Assembly.order(promoted: :desc)`

Reply:
The point is to automatically pickup any changes/improvements/fixes if the query changes in core. But if it's not easy to do or desired we can skip it.